tools/shared.py

```diff
@@ -29,9 +29,14 @@
 # Configure logging before importing any other local modules so even
 # log message during import are shown as expected.
 DEBUG = int(os.environ.get('EMCC_DEBUG', '0'))
+EMCC_LOGGING = int(os.environ.get('EMCC_LOGGING', '1'))
+log_level = logging.ERROR
```

Comment:
should this be WARN by default? (what was the default before? do we actually use WARN?)

Reply:
EMCC_LOGGING default to 1, so the default here is, and was before, logging.INFO.

`logging.ERROR` is the new level introduced here that is only used if you explicitly set `EMCC_LOGGING=0`.